src/multi_vector_simulator/C0_data_processing.py

```diff
@@ -1473,24 +1473,24 @@ def receive_timeseries_from_csv(
     file_path = os.path.join(settings[PATH_INPUT_FOLDER], TIME_SERIES, file_name)
     C1.lookup_file(file_path, dict_asset[LABEL])
 
-    data_set = pd.read_csv(file_path, sep=",")
+    data_set = pd.read_csv(file_path, sep=",", keep_default_na=True)
 
     if FILENAME in dict_asset:
         header = data_set.columns[0]
 
     if len(data_set.index) == settings[PERIODS]:
         if input_type == "input":
-            dict_asset.update(
-                {
-                    TIMESERIES: pd.Series(
-                        data_set[header].values, index=settings[TIME_INDEX]
-                    )
-                }
+            timeseries = pd.Series(data_set[header].values, index=settings[TIME_INDEX])
```

Comment:
```suggestion
            timeseries = pd.Series(data_set[header].values, index=settings[TIME_INDEX], name=dict_asset[LABEL])
```

Reply:
Why do you propose this? It was not in the original code...